compute/snapshots/snapshots_test.go

```diff
@@ -145,6 +145,26 @@ func TestComputeSnapshotsSnippets(t *testing.T) {
 		t.Errorf("createSnapshot got %q, want %q", got, want)
 	}
 
+	buf.Reset()
+	want = snapshotName
+
+	if err := listSnapshots(buf, tc.ProjectID, ""); err != nil {
```

Comment:
I think the typical practice elsewhere is 1 test case per region tag/sample.

Reply:
There is no single convention, it varies based on the needs of given tests. It would do more harm than good in this case.

There's a lot of initialization that needs to be done up front in this file (see lines 123-132), which creates a lot of boilerplate, which would have to be duplicated if I split this up into separate tests. Besides, some steps rely on previous steps to be complete (creation must come before listing, listing must come before deletion) so it wouldn't be "one testcase = one snippet" anyway.

Splitting up the test function would make sense if we were looking to parallelize the tests (I've done this for another product already), but these tests execute quickly enough that this is not a concern.

Reply:
Nevermind, I take that back. Done.